add PybNano V2 baord

shaoziyang opened the pull request.

shaoziyang committed:
add PybNano V2 baord

mmoskal commented on boards/pybnanov2/board.h:
```diff
@@ -0,0 +1,38 @@
+    CFG_PIN_BTN_A, 0x2D, // PC_13
+    204, 0x80000, // FLASH_BYTES = 0x80000
+    205, 0x18000, // RAM_BYTES = 0x18000
+    208, 0x1f884ff3, // BOOTLOADER_BOARD_ID = 0x1f884ff3
```
Please use a unique device identifier. You can generate a random number with the link at the bottom of this site: https://microsoft.github.io/uf2/patcher/

mmoskal reviewed (changes requested):
looks good, modulo the board id

shaoziyang committed:
change BOARD_ID

use new random number from https://microsoft.github.io/uf2/patcher/ for BOOTLOADER_BOARD_ID

mmoskal commented:
Thank you!
